packages/react-devtools-shared/src/symbolicateSource.js

```diff
@@ -68,9 +68,9 @@ async function symbolicateSource(
         resourceLine.length,
       );
 
-      const sourceMap = await fetchFileWithCaching(sourceMapURL).catch(
-        () => null,
-      );
+      const sourceMap = await fetchFileWithCaching(
+        new URL(sourceMapURL, sourceURL).toString(),
+      ).catch(() => null);
```

Comment:
Does this work with inlined source maps?

Reply:
Yes.

```js
new URL('data:application/json;base64,data', 'http://127.0.0.1/').toString()
"data:application/json;base64,data" 
```